### ISSUE TITLE
Missing admin screen headings

### ISSUE THREAD

**Issue by danieliser** [PopupMaker/Popup-Maker#778]
@ivanlutrov  Reported that multiple admin screens under our control are missing `H1` tags.

https://wordpress.org/support/topic/missing-admin-screen-headings/

This is an easy accessibility win.

> These admin screens have tabs but don’t have a page heading:
> 
> * Popup maker > All popups
> * Popup maker > Popup themes
> * Popup maker > Extend
> 
> These admin screens have tabs but do have a page heading:
> 
> * Popup maker > Tools

**Comment by fpcorso** [comment]
@danieliser Added H1's for all of these pages:
![image](https://user-images.githubusercontent.com/9730040/83670329-fd39de80-a5a0-11ea-9426-ce542a7721af.png)

**Comment by danieliser** [comment]
@fpcorso nice.
